README.md

```diff
@@ -197,6 +197,26 @@ reaction app.onPinChange(uint8_t pin, react_callback cb);
 
 Execute a callback when a changing voltage is detected on a pin. Make sure the pin provided is an interrupt pin.
 
+```cpp
+reaction app.onPinRisingNoInt(uint8_t pin, react_callback cb);
+```
+
+Same as `onPinRising`, but without using interrupts. Useful for attaching a change callback to a pin 
+not triggering interrupts.
```

Comment:
New lines have significance in markdown and this might render strangely. Usually best to keep everything on one line unless you're starting a new paragraph.